Source/SwiftLintFramework/Rules/Style/DeinitRequiredRule.swift

```diff
@@ -0,0 +1,76 @@
+import SourceKittenFramework
+
+public struct DeinitRequiredRule: OptInRule, ConfigurationProviderRule, AutomaticTestableRule {
```

Comment:
what do you think about renaming it to `RequiredDeinitRule`? It'd be more in line with the existing `RequiredEnumCaseRule`.